### DISCUSSION TITLE
How to have operators highlighted in different color like Sublime Text?

### DISCUSSION THREAD

**Discussion by ghost** [orbitalquark/textadept#359] [Q&A] [answered]
Currently they are in the same color as variables, functions,...

**Comment by orbitalquark** [comment, marked as answer]
You would want to change your theme to set styles for `lexer.OPERATOR`. 
By default it is unset: 
https://github.com/orbitalquark/textadept/blob/96f58f1fa34fa50feead22df21b34412c1327da1/themes/light.lua#L62.

You can copy one of Textadept's default themes to your 
*~/.textadept/themes/* directory, change that property, and then call 
`ui.set_theme()` from your *~/.textadept/init.lua* with that theme's 
name. If it's the default 'light' theme, then you don't have to do 
anything; Textadept uses it automatically.
